backend/open_webui/test/apps/webui/retrieval/test_dimension_adaptation.py

```
@@ -0,0 +1,366 @@
+            test_vector = [0.1] * 384
+            test_item = {
+                "id": str(uuid.uuid4()),
+                "text": "This is a test document for multi-tenant collection.",
+                "vector": test_vector,
+                "metadata": {"source": "test", "dimension": 384}
+            }
+            
+            try:
+                # Insert item with tenant ID
+                self.vector_db_client.insert_with_tenant(
+                    collection_name=test_collection_name,
```

Ruff (0.8.2)

Undefined name `test_collection_name`

(F821)